migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -553,6 +553,7 @@ public void saveRuns(final Collection<Run> runs) throws IOException {
       Run run = serviceManager.getRunDao().getByAlias(newRun.getAlias());
       if (run == null) {
         run = newRun;
+        run.getStatus().setLastUpdated(new Date());
```

Comment:
`Status` is about to disappear, so it doesn't really matter, but should use `timeStamp` member var of DefaultMigrationTarget instead of new `Date`s so everything with the migration's date has the exact same value